testsuite/podman_runner/03_run_controller.sh

```diff
@@ -2,6 +2,6 @@
 set -ex
 src_dir=$(cd $(dirname "$0")/../.. && pwd -P)
 
-podman run --rm -d --network uyuni-network-1 -v /tmp/test-all-in-one:/tmp --name controller-test -h controller-test -v ${src_dir}/testsuite:/testsuite ghcr.io/$UYUNI_PROJECT/uyuni/ci-test-controller-dev:$UYUNI_VERSION
+sudo -i podman run --rm -d --network uyuni-network-1 -v /tmp/test-all-in-one:/tmp --name controller-test -h controller-test -v ${src_dir}/testsuite:/testsuite ghcr.io/$UYUNI_PROJECT/uyuni/ci-test-controller-dev:$UYUNI_VERSION
```

Comment:
Not sure about it. But maybe is clear no run the scripts (.sh files) using sudo, so you don't need to worry inside the script.

Reply:
Then I would need to put the sudo in each step of the github action..I thought about it , too.